### PR TITLE
Fixed the hidden lobby bug

### PR DESCRIPTION
**Pull request by kwonghua**
empty lobby message will now display correctly to users in the case of all hidden lobbies.

### CODE REVIEW

File: static/js/pages/home.js [hunk 1]
```diff
@@ -98,6 +98,11 @@ var app = new Vue({
         this.activePrompts = prompts.filter(p => !p.rated);
 
         this.lobbies = await getUserLobby(this.loggedIn);
+        for(let lobby of this.lobbies){
```
Comment (bricehalder) [inline static/js/pages/home.js:101]: If you want to modernize / functionalize this you can do something like 
```
this.hiddenLobbies = this.lobbies.reduce((count, lobby) => count + lobby.isHidden, 0);
```

**Review by bricehalder — APPROVED** [review]
Nice, thanks for quickly fixing this!!!

File: static/js/pages/home.js (hunk 1 above)
Comment (bricehalder) [reply static/js/pages/home.js:101]: If you want to make the change and commit it feel free and then we can merge this. Here is some reading on map, reduce, and filter in JS

Comment (kwonghua) [reply static/js/pages/home.js]: Gotcha, thanks for that link too. I went ahead and made the changes suggested

**Pull request merged**